checkbox/index.js

```diff
@@ -0,0 +1,43 @@
+module.exports = Checkbox;
+
+/**
+ * Creates a snackbar component. It is possible to pass a global options object as an attribute. For public methods, see ./actions.
+ */
+function Checkbox() {}
+
+Checkbox.prototype.view = __dirname + '/views';
+Checkbox.prototype.style = __dirname + '/styles';
+Checkbox.prototype.name = 'checkbox';
+Checkbox.prototype.components = [];
+
+require('./operations');
+require('./actions');
+
+Checkbox.prototype.init = function(model) {
+  this.groupHeader = this.getAttribute('groupHeader');
+  
+  var self = this;
+  if(this.groupHeader) {
+    model.start('undecided', 'group', function (group) {
```

Comment:
Creative/interesting solution. I'd probably not thought of doing it this way, but I think it fits well to the use case.